libnopegl/src/utils.h

```diff
@@ -57,6 +57,8 @@
 
 #define NGLI_STATIC_ASSERT(id, c) typedef char ngli_checking_##id[(c) ? 1 : -1]
 
+#define NGLI_FIELD_SIZEOF(name, field) (sizeof(((struct name *)0)->field))
```

Comment:
I'm really not a fan of having the struct here tbh, I'd rather be explicit in the caller

Reply:
Updated.